CMakeLists.txt

```diff
@@ -19,6 +19,7 @@
 
 CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
 
+SET(CMAKE_CXX_STANDARD 11)
```

Comment:
If you don't want to enforce this (because that means you should enforce it on all dependant packages, otherwise they won't work), you can replace `isfinite(T)` below by `T <= std::numeric_limits<value_type>::max()`.

Reply:
This is two unrelated issue, the c++ 11 is required for some changes in the continuous validation that do not compile without : 

```
In file included from /local/dev_hpp/src/hpp-core/src/continuous-validation.cc:19:0:
/local/dev_hpp/src/hpp-core/include/hpp/core/continuous-validation.hh: In instantiation of ‘void std::vector<_Tp, _Alloc>::_M_insert_aux(std::vector<_Tp, _Alloc>::iterator, const _Tp&) [with _Tp = hpp::core::ContinuousValidation::AddObstacle; _Alloc = std::allocator<hpp::core::ContinuousValidation::AddObstacle>; std::vector<_Tp, _Alloc>::iterator = __gnu_cxx::__normal_iterator<hpp::core::ContinuousValidation::AddObstacle*, std::vector<hpp::core::ContinuousValidation::AddObstacle> >; typename std::_Vector_base<_Tp, _Alloc>::pointer = hpp::core::ContinuousValidation::AddObstacle*]’:
/usr/include/c++/5/bits/stl_vector.h:925:17:   required from ‘void std::vector<_Tp, _Alloc>::push_back(const value_type&) [with _Tp = hpp::core::ContinuousValidation::AddObstacle; _Alloc = std::allocator<hpp::core::ContinuousValidation::AddObstacle>; std::vector<_Tp, _Alloc>::value_type = hpp::core::ContinuousValidation::AddObstacle]’
/local/dev_hpp/src/hpp-core/src/continuous-validation.cc:342:38:   required from here
/local/dev_hpp/src/hpp-core/include/hpp/core/continuous-validation.hh:86:13: error: non-static reference member ‘hpp::core::ContinuousValidation& hpp::core::ContinuousValidation::AddObstacle::owner_’, can’t use default assignment operator
       class AddObstacle
             ^
In file included from /usr/include/c++/5/vector:69:0,
                 from /local/dev_hpp/install/include/hpp/pinocchio/pool.hh:23,
                 from /local/dev_hpp/src/hpp-core/include/hpp/core/continuous-validation.hh:22,
                 from /local/dev_hpp/src/hpp-core/src/continuous-validation.cc:19:
/usr/include/c++/5/bits/vector.tcc:343:16: note: synthesized method ‘hpp::core::ContinuousValidation::AddObstacle& hpp::core::ContinuousValidation::AddObstacle::operator=(const hpp::core::ContinuousValidation::AddObstacle&)’ first required here 
    *__position = __x_copy;
```

Reply:
This leads to a more general question: do we move to C++11 ? This cannot be decided at hpp-core level.

For this specific problem, and if we decide to keep C++98 compatibility, the workaround is to replace hpp::core::ContinuousValidation::AddObstacle::owner_ by a pointer.

Reply:
Using `SET(CMAKE_CXX_STANDARD 11)` is not that nice, also because this means that we won't test newer versions of the standard, that are provided by default on new compilers / OS.

So this won't force use to drop all the dynamic exception declarations, while we should probably discuss about that.

Would your workaround require a lot of work ?

Reply:
> Would your workaround require a lot of work ?

No definitely. @florent-lamiraux can answer this more precisely.

Reply:
I suggest to just merge this for now. We can find a better solution later :)

Reply:
You have then to make sure that everythin below is compiled in C++ 11.

Reply:
True. 

I'll add ` SET(CMAKE_CXX_STANDARD 11)` in the Config.cmake then.